packages/server/src/index.ts

```diff
@@ -21,7 +21,11 @@
 import app from './server';
 
 async function main() {
-  await app.listen({ port: process.env.SERVER_PORT || 7546 });
+  const server = await app.listen({ port: process.env.SERVER_PORT || 7546 });
+
+  // set request timeout to ensure sockets are closed after specified time of inactivity
+  const requestTimeoutMs = parseInt(process.env.SERVER_REQUEST_TIMEOUT_MS || '60000');
+  server.setTimeout(requestTimeoutMs);
```

Comment:
The setTimeout method allows to send a callback function that will be executed in case a timeout is executed:
![image](https://github.com/hashgraph/hedera-json-rpc-relay/assets/123040664/6082036a-c6c7-468d-855b-2196af677d36)


I think we should use it to capture metrics of timeouts.

however, we don't have the registry yet in here... 


so I was thinking if we can change the approach and do something like this??

I haven't tested this yet, but basically uses:
https://www.npmjs.com/package/koa-timeout


and move it to file `Server.ts`

and do it something like this:
```
// Custom middleware to increment the metric on timeout
app.use(async (ctx, next) => {
  try {
    await next();
  } catch (err) {
    if (err.message === 'Request timeout') {
      // Increment the timeout metric here
      // For example, using a metric library like Prometheus:
      // prometheusClient.counter('request_timeouts_total').inc();
      console.log('Request timeout reached');
    }
    throw err;
  }
});

// Apply the timeout middleware
app.use(timeout(5000)); // Set the timeout value in milliseconds

```